src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -367,7 +137,13 @@ class Breakpoints extends PureComponent<Props> {
         })}
         key={locationId}
         onClick={() => this.selectBreakpoint(breakpoint)}
-        onContextMenu={e => this.showContextMenu(e, breakpoint)}
+        onContextMenu={e => {
+          const menuProps = Object.assign({}, this.props, {
+            breakpoint,
+            contextMenuEvent: e
+          });
+          showContextMenu(menuProps);
```

Comment:
I think this can be `onContextMenu={e => showContextMenu({...this.props, breakpoint, contextMenuEvent: e })}`